Resources/Prototypes/_Sunrise/FleshCult/mobs.yml

```
@@ -610,0 +1,609 @@
+﻿- type: entity
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)